Mathlib/RingTheory/PowerSeries/Basic.lean

```diff
@@ -261,6 +265,16 @@ theorem coeff_ne_zero_C {a : R} {n : ℕ} (h : n ≠ 0) : coeff R n (C R a) = 0
 theorem coeff_succ_C {a : R} {n : ℕ} : coeff R (n + 1) (C R a) = 0 :=
   coeff_ne_zero_C n.succ_ne_zero
 
+theorem C_injective : Function.Injective (C R) := by
+  intro a b H
+  have := (ext_iff (φ := C R a) (ψ := C R b)).mp H 0
+  rwa [coeff_zero_C, coeff_zero_C] at this
+
+protected theorem subsingleton_iff : Subsingleton R ↔ Subsingleton R⟦X⟧ := by
```

Comment:
Since the full name is `PowerSeries.subsingleton_iff` I think you should switch the two implications.

Reply:
Done